goalie/mesh_seq.py

```diff
@@ -8,11 +8,12 @@
 from firedrake.petsc import PETSc
 from firedrake.pyplot import triplot
 from .function_data import ForwardSolutionData
-from .interpolation import transfer
 from .log import pyrint, debug, warning, info, logger, DEBUG
 from .options import AdaptParameters
+from animate.interpolation import transfer
 from animate.quality import QualityMeasure
-from .utility import AttrDict, Mesh
+from animate.utility import Mesh
+from .utility import AttrDict
```

Comment:
@jwallwork23 I saw now that `.utility` is separated from other relative imports. And imports are a bit disordered in general. 

Perhaps we should (in a separate issue) organise imports manually or e.g. add [isort](https://pycqa.github.io/isort/index.html) to pre-commit?

Reply:
Good idea. See #162.